app/controllers/concerns/sortable.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Sortable 
+  extend ActiveSupport::Concern
+
+  private
+
+  def sort_with_params(items, params)
+    items = items.order_by_date(params[:order]) if params[:order].present?
+    items = items.order_by_price(params[:price]) if params[:price].present?
+    items
+  end
```

Comment:
I think it doesn't matter (I mean two parameters for filtering) because the final order is determined by the last filtering. My propose 
`scope :order_by, ->(attribute, type = :desc) { reorder(attribute => type) if attribute.present? }
items.order_by(params[:field], params[:type_order])`

Reply:
You're misunderstanding it's not a filter, that a sort https://i.imgur.com/hZr2WpH.png

Reply:
okay, I did mistake in my description, but in my function I did order(sort). And we have this point 
![image](https://github.com/oleksiimikhno/book-store/assets/41300309/22209fec-d943-454a-8f03-c97f17e7713d)

and I thought we could get a special word like price, updated_at, name and a second parameter like desc or asc and one function that will take two parameters and do the sorting
![image](https://github.com/oleksiimikhno/book-store/assets/41300309/da2bf80d-f348-420f-b097-7ff40f5a5d39)